src/rax.c

```diff
@@ -728,6 +733,7 @@ int raxGenericInsert(rax *rax, unsigned char *s, size_t len, void *data, void **
             return 0;
         }
         splitnode->data[0] = h->data[j];
+        rax->alloc += rax_alloc_size(splitnode);
```

Comment:
i'd like to run some benchmarks and see what's the effect of all these calls on the performance.